src/web/components/ClientSideTokenGeneration/CstgDomainsTable.scss

```diff
@@ -0,0 +1,47 @@
+.cstg-domains-management {
+  .cstg-domains-table {
+    width: 100%;
+    th:first-of-type,
+    td:first-of-type {
+      padding-left: 10px;
+      width: 21px;
+    }
+
+    th.action {
+      text-align: end;
+    }
+
+    tr {
+      .tristate-checkbox {
+        margin-right: 16px;
+      }
+
+      button[role='checkbox'] {
```

Comment:
I've seen this styling before. Is it worth looking into extracting this out to somewhere more global?

Reply:
make sense to be global, updated